database.py

```diff
@@ -88,6 +88,11 @@ def get_recent(self):
         self.cur.execute(sql)
         return self.cur.fetchall()
 
+    def get_one(self, url):
```

Comment:
Since this is looking up the `real_link`, can you change the name of the method to `query_real_link`.

And rename `query` to `query_micro_link`.